src/components/switch/bl-switch.stories.mdx

```diff
@@ -93,6 +99,16 @@ Disabled state can be set via `disabled` attribute. A switch can be `disabled` a
   </Story>
 </Canvas>
 
+## Keyboard navigation
+
+Switch component handles keyboard navigation and highlights itself while navigating with keyboard. First `Tab` focuses the element and user can turn on and turn off the switch with `Enter` or `Space` keys.
+
+<Canvas>
+  <Story name="Keyboard navigation" parameters={{ extraPaddingForChromatic: true }} play={focusSwitch}>
+    {SwitchTemplate.bind({})}
+  </Story>
```

Comment:
I think this doesn't work now. In chromatic screenshot, I don't see focus state.